theano/tensor/basic.py

```diff
@@ -3139,7 +3141,9 @@ def var(input, axis=None, keepdims=False):
     centered_input = input - mean_input
 
     # return the mean sqr
-    v = mean((centered_input ** 2), axis, keepdims=keepdims)
+    #divisor = theano.tensor.cast(prod(input.shape[axis]) - ddof, theano.config.floatX)
+    divisor = prod(input.shape[axis]) - ddof
```

Comment:
Also, for consistency and prevent strange difference, we are better to handle this the same way mean is doing:

    for i in axis:
        s = true_div(s, shp[i])

Or we should change mean to use the same way of doing it. The advantage of the loop, is if there is only 1 axis, we build a smaller graph. (but I think opt will clean it up).